src/main/kotlin/lotto/domain/data/Money.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.domain.data
+
+data class Money(
+    private val value: Int
+) {
+    init {
+        require(value > 0) { "돈이 이상해요 입력값: $value" }
+    }
+
+    fun getValue(): Int {
+        return this.value
+    }
+}
```

Comment:
getValue가 필요하다면
```suggestion
data class Money(
    val value: Int
) {
    init {
        require(value > 0) { "돈이 이상해요 입력값: $value" }
    }
}
```
자바스타일 보다는 코틀린 프로퍼티를 이용해보자